smoke/IotEdgeQuickstart/details/IotedgedLinux.cs

```diff
@@ -65,20 +65,57 @@ static string GetIpAddress()
         }
     }
 
+    public class UriSocks
+    {
+        const int ManagementPort = 15580;
+        const int WorkloadPort = 15581;
+
+        public UriSocks()
```

Comment:
Combine this constructor with the other one.  Set to default value if it doesn't provide,

Reply:
Makes sense.